components/Timeline.jsx

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable @next/next/no-img-element */
```

Comment:
Let's remove this from all places now and enforce it so we always do this blur thing?

Reply:
Will do

Reply:
I can't get the Next Image component to work the way I want for a couple SVGs. Keeping those as imgs. Merging anyways.